app/i18n/ja/people.ts

```diff
@@ -168,6 +184,7 @@ const people: People = {
           'CEO - The Brand Garden',
           'Henleyビジネススクール - 組織リーダーシップ修士号',
         ],
+        link: 'https://www.linkedin.com/in/richardnall/?originalSubdomain=uk',
```

Comment:
replace with `https://www.linkedin.com/in/richardnall` for consistency. Double check the other `/people.ts` files as well.

Reply:
ok done